RELEASES.md

```diff
@@ -0,0 +1,77 @@
+# Grafana Releases
+
+## Release schedule
+
+Release cadence of first beta-release being cut is 6 weeks.
+
+## Branch management and versioning strategy
+
+We use [Semantic Versioning](https://semver.org/).
+
+We maintain a separate branch for each minor release, named `v<major>.<minor>.x`, e.g. v6.0.x, v6.5.x. The release branch is created from the master branch just before  releasing the first beta-release of a minor version, e.g. v6.5.0-beta1.
+
+New features, bug fixes and changes are squashed and merged into the master branch. Bug fixes needed to be included in a patch release are being cherry-picked from the master branch. The master branch is the development branch.
+
+## Release management roles
+
+### Release coordinator
```

Comment:
This is confusing. The first heading under Release management roles is an actual role title, but the second one does not look like a role title. Please make consistent.

Reply:
Also, I suggest capitalizing the roles so it is clear throughout that these are roles or titles.

Release Coordinator
Release Responsible

Reply:
Yeah I know. Tried to come up with a good role name of the one doing the actual release. This was my best alternative :) Release manager feels like something else, right?

Do you have a better suggestion?

Reply:
Release Executor? That sounds all badass. I will see if anyone on Slack has ideas.